frontend/src/container/Dashboard.jsx

```diff
@@ -130,6 +130,7 @@ class Dashboard extends React.Component {
         if (props.appList.some(a => a.slug === 'agenda') && fetchWorkspaceDetail.json.agenda_enabled) {
           this.loadCalendarDetail()
         }
+        GLOBAL_dispatchEvent({ type: CUSTOM_EVENT.SET_TITLE, data: { title: fetchWorkspaceDetail.json.label } })
```

Comment:
shouldn't you also handle the event ALL_APP_CHANGE_LANGUAGE in this file ?

Reply:
Not in this case because I only display the label of the current workspace, but we still need to discuss about which title should be for all the page, (e.g in dashboard `Dashbord {{workspaceLabel}}` and `ContentList {{workspaceLabel}}`)

Reply:
ok for ALL_APP_CHANGE_LANGUAGE.

For the list of titles, prepare a document with the exhaustive list of pages that requires a title change with the title you think would fit for each